app/src/main/java/com/hedvig/app/service/LoginStatusService.kt

```diff
@@ -1,44 +1,52 @@
 package com.hedvig.app.service
 
 import android.content.Context
+import android.content.SharedPreferences
 import com.apollographql.apollo.ApolloClient
 import com.apollographql.apollo.coroutines.await
 import com.hedvig.android.owldroid.graphql.ContractStatusQuery
 import com.hedvig.app.util.extensions.getAuthenticationToken
-import com.hedvig.app.util.extensions.getStoredBoolean
-import com.hedvig.app.util.extensions.isLoggedIn
-import com.hedvig.app.util.extensions.setIsLoggedIn
+
+private const val SHARED_PREFERENCE_IS_LOGGED_IN = "shared_preference_is_logged_in"
```

Comment:
Should we put these in the `companion object`?

Reply:
I think it makes sense to have top level here since we only have one class in the file, feels unnecessary to create companion object overhead

Reply:
My preference isn't super-big towards `companion object`s, the only thing that I think talks particularly in favor of `companion object`s are that Android Studio lists the file as `LoginStatusService` instead of `LoginStatusService.kt` when there's only one class in the file 😅 

What kind of overhead are you talking about for the `companion object`? The added lines, or something the compiler generates? 🤔

Reply:
Thats a good point :) I was thinking both the added lines and the added java code generated by the compiler, but lets go with companion objects for now since that's what we're used to

Reply:
Out of curiousity, I checked what the compiler generated for either case (`companion object` vs file-level `const`s) - they generate more or less the same code, actually - file-level `const`s will generate something like this:

```kotlin
static class ${FileName}Kt {
    private static final CONST_VARIABLE = "CONST_VARIABLE";
}
```

while `companion object` generates something like this:

```kotlin

class FileNameKt { // Parent class
    static class Companion {
        private static final CONST_VARIABLE = "CONST_VARIABLE";
    }
}
```

Reply:
Cool!